llama-index-packs/llama-index-packs-corrective-rag/llama_index/packs/corrective_rag/base.py

```diff
@@ -0,0 +1,134 @@
+"""Corrective RAG LlamaPack class."""
+
+from typing import Any, Dict, List
+
+from llama_index.core import VectorStoreIndex, SummaryIndex
+from llama_index.core.llama_pack.base import BaseLlamaPack
+from llama_index.llms.openai import OpenAI
+from llama_index.core.schema import Document, NodeWithScore
+from llama_index.core.query_pipeline.query import QueryPipeline
+from llama_index.tools.tavily_research.base import TavilyToolSpec
+from llama_index.core.prompts import PromptTemplate
```

Comment:
nit: i wonder if our linters are working? This should be sorted?

Reply:
todo: fix linters 😢

Reply:
Hehe that's on CI/CD side, right? I think we can also still make use of pre-commits? And, our pre-commits have changed since v0.9.xx.

@ravi03071991 have you re-installed the pre-commits?

```
pre-commit install && git ls-files | xargs pre-commit run --show-diff-on-failure --files
```

Reply:
checking it.